package/bin/edges_phones_helper.py

```diff
@@ -8,6 +8,7 @@
 from splunklib import modularinput as smi
 
 from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
To keep code consistency and avoid adding another dependency, `timedelta` can be used instead of `relativedelta`
```suggestion
from datetime import datetime, timedelta
```